auth-sample/build.gradle

```diff
@@ -67,9 +67,10 @@ android {
 }
 
 dependencies {
-//    implementation files('../auth-lib/build/outputs/aar/auth-release.aar')
-    implementation 'com.spotify.android:auth:2.0.2'
+//    implementation files('../auth-lib/build/outputs/aar/auth-store.aar')
+    implementation 'com.spotify.android:auth:2.1.0'
 
+    implementation 'androidx.browser:browser:1.4.0'
     implementation 'androidx.appcompat:appcompat:1.1.0'
     implementation 'com.google.android.material:material:1.0.0'
     implementation 'com.squareup.okhttp3:okhttp:3.12.3'
```

Comment:
Consider updating the dependency, so the implementor will not use unsafe code: https://security.snyk.io/package/maven/com.squareup.okhttp3:okhttp